### ISSUE TITLE
Ability to specify target view to add constraints

### ISSUE THREAD

**Issue by d6u** [SnapKit/SnapKit#140]
If I do
```swift
        self.contentView.snp_updateConstraints { make in
            make.height.equalTo(100)
        }

        println(self.constraints()) // => [<<SnapKit.LayoutConstraint:0x7fb0da5fa480> <UITableViewCellContentView:0x7fb0da5fac10>.height == 100.0>]
        println(contentView.constraints()) // => []
```
The height constraint will be added to `self` instead of `self.contentView`

But if I do the original way:
```swift
        self.contentView.setTranslatesAutoresizingMaskIntoConstraints(false)
        self.contentView.addConstraint(NSLayoutConstraint(
            item: self.contentView,
            attribute: NSLayoutAttribute.Height,
            relatedBy: NSLayoutRelation.Equal,
            toItem: nil,
            attribute: NSLayoutAttribute.NotAnAttribute,
            multiplier: 1,
            constant: CGFloat(20 * index)))

        println(self.constraints()) // => []
        println(contentView.constraints()) // => [<NSLayoutConstraint:0x7fdac34b57a0 V:[UITableViewCellContentView:0x7fdac34b3990(0)]>]
```
Which is the correct behavior I want.

Is there anyway to achieve that?

**Comment by robertjpayne** [comment]
@d6u what version of SnapKit are you using? I'm pretty sure I resolved this by ensuring static width/height constraints are added directly on the view they were created for.

**Comment by d6u** [comment]
Sorry, I was on 0.12. I confirm that it is already fixed in 0.13. Thanks!

**Comment by robertjpayne** [comment]
No worries @d6u thanks for reporting anyways :+1:
